### ISSUE TITLE
Updating preventSwipe array prop after initial render does not take effect.

### ISSUE THREAD

**Issue by Ashutosh-Roxiler** [3DJakob/react-tinder-card#88]
Thats why Which causes UI distortion bec user can swipe very fast also again and again towards right only this issue occurs on mobile browsers only. #31 also trying to get rid of this problem.

https://user-images.githubusercontent.com/85158856/155265845-602aa16e-762b-470f-8f88-199a6c5aa744.mp4

**Comment by LinusU** [comment]
I think that the problem with the cards being visible outside of the screen could be fixed by using `overflow: hidden;` on the container?

**Comment by 3DJakob** [comment]
Yes exactly like Linus said just add `overflow: hidden;` on your parent container like the `app` container and it will work 👍
